File: web/src/model/victron_controller/dashboard/v0_2_0/from_0_1_0_wsservermessage_snapshot.ts
// @ts-nocheck
import {Snapshot as v0_2_0_Snapshot} from './WsServerMessage'
import {Snapshot as v0_1_0_Snapshot} from '../v0_1_0/WsServerMessage'

export function convert__ws_server_message__snapshot__from__0_1_0(from: v0_1_0_Snapshot): v0_2_0_Snapshot {
    return new v0_2_0_Snapshot (
        JSON.parse(JSON.stringify(from.body))
    )
}

File: crates/dashboard-model/src/victron_controller/dashboard/v0_2_0/from_0_1_0_wsservermessage_snapshot.rs


pub fn convert__wsservermessage__snapshot__from__0_1_0(from: &crate::victron_controller::dashboard::v0_1_0::ws_server_message::Snapshot) -> crate::victron_controller::dashboard::v0_2_0::ws_server_message::Snapshot {
    crate::victron_controller::dashboard::v0_2_0::ws_server_message::Snapshot {
        body: serde_json::from_value(serde_json::to_value(&from.body).unwrap()).unwrap(),
    }
}